package.lisp

```diff
@@ -23,3 +23,35 @@
 
 (defpackage :stumpwm-user
   (:use :cl :stumpwm))
+
+(defpackage :stumpui
+  (:use :cl)
+  (:export
+   ;; Window protocol
```

Comment:
I mean protocol in the sense that it is a set of generic functions defining what needs to be implemented. This is fairly common usage in CL, think of the MetaObject Protocol as another example.